azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -626,6 +622,14 @@ private void doRun() {
     writeStatus();
   }
 
+  private void uploadExecutableNode() {
+    try {
+      this.loader.uploadExecutableNode(this.node, this.props);
+    } catch (final ExecutorManagerException e1) {
```

Comment:
Although it makes me wonder does it make sense to continue at all if this fails? Maybe throw an exception instead?